compute/errors.go

```diff
@@ -24,7 +24,7 @@ import (
 	"github.com/onflow/cadence/runtime"
 	"github.com/onflow/cadence/runtime/ast"
 	runtimeErrors "github.com/onflow/cadence/runtime/errors"
-	"github.com/onflow/flow-go/fvm"
+	errors2 "github.com/onflow/flow-go/fvm/errors"
```

Comment:
Maybe this would be better:
```suggestion
	fvmErrors "github.com/onflow/flow-go/fvm/errors"
```

Reply:
👍 improved in 898e9bb